tests/mmvdump.rs

```diff
@@ -0,0 +1,43 @@
+use std::process::{Command, Stdio};
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::path::PathBuf;
+
+#[test]
+fn test_mmvdump() {
+    let crate_dir = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
+
+    let mut mmvdump_path = crate_dir.clone();
+    mmvdump_path.push("target/debug/mmvdump");
```

Comment:
don't do this, add a function to the library to export the string dump. Ideally `print_indoms` would be `write_indoms` taking `io::BufWriter` as a parameter